src/client/components/Radio/ButtonRadio.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const users = ['Student', 'Mentor', 'Admin'];
+function ButtonRadio() {
+  return (
+    <div className="btn-group">
+      {users.map((u) => (
```

Comment:
Hi @daxiousm I would suggest you change the **u** here to **user** so you have something like 
{users.map((user) => (